scalableminds/webknossos
# Use full prefix in sitemap urls

### URL of deployed dev instance (used for testing):
- https://fixsitemap.webknossos.xyz

### Steps to test:
- visit `/sitemap.xml`
- test also on localhost
- urls should be correct

------
